ci/build.yml

```
@@ -0,0 +1,21 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 
```

This should have a specific version. We currently work from the 1.13 branch of Go so this should probably be… 
```suggestion
    tag: 1.13.15
```